_posts/2022-12-23-trino-summit-2022-upsolver-recap.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: post
+title: "Using Trino to analyze a product-led growth (PLG) user activation funnel"
+author: "Mei Long, Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-summit-2022/upsolver.jpg
+---
+
+As the holiday season approaches, we have reached the end of our
+[Trino Summit 2022 recap posts]({% post_url 2022-11-21-trino-summit-2022-recap %}).
+With the last talk of the summit, Mei Long from Upsolver gave an insightful
+overview of how they use data to inform product decisions.
+
+<!--more-->
+
+{% youtube MCB_1furnAo %}
+
+<a class="btn btn-pink btn-md" target="_blank" href="/assets/blog/trino-summit-2022/Trino@Upsolver.pdf">
+  Check out the slides!
+</a>
+
+## Recap
+
+When talking about product-led growth, it helps to start by defining what it
+even means. The core idea is simple: see how users engage with your product,
+and make decisions based on how you can improve the product to better serve
+those users. At Upsolver, the goal is to maximize user value. The issue is that
+while this can be simple in some situations, when you're delivering complicated
+analytics tools, it's not always immediately clear what features would be the
+most valuable or useful. You need a lot of data to glean a lot of insight, and
+you need to make sure your insights that can lead to action. And of course, you
+need to be absolutely certain that your data is high-quality, accurate, and can
+be trusted, lest you end up accidentally giving a customer a $10 million 
+discount.
+
+Mei explores the initial pass at using analytics to drive PLG at Upsolver,
+letting her intern use a tool called Amplitude that worked for a time and for
+limited use cases. As Upsolver grew, the analytics requirements did, too, and
+Amplitude wasn't powerful enough for the use case Upsolver needed, nor for the
+more complicated queries and analysis that needed to be run.
+
+Want to guess what query engine they swapped to using? Trino. And with a quick
```

Comment:
I don't like this turn of phrase here, personally, and would just say something flat like "After weighing their options, Upsolver switched their query engine to Trino." But I'm not gonna throw a fit over it.

Reply:
I'm gonna keep it this way because it's meant to be a little cheeky and fun (why would we be writing about it if it weren't Trino), which is the tone we're going for in a blog post.